packages/lab/src/__tests__/__e2e__/slider/Slider.cy.tsx

```diff
@@ -1,202 +1,79 @@
-    const changeSpy = cy.stub().as("changeSpy");
-
-    cy.mount(
-      <Slider
-        label={"TestLabel"}
-        min={-8}
-        max={8}
-        step={1}
-        defaultValue={[-1, 3, 7]}
-        pushable
-        pushDistance={3}
-        onChange={changeSpy}
-      />
-    );
-
-    cy.findAllByRole("slider").should("have.length", 3);
-
-    cy.findAllByRole("slider").eq(0).focus();
-    cy.realPress("ArrowRight");
-    cy.get("@changeSpy").should("have.been.calledWith", [0, 3, 7]);
-
-    cy.realPress("ArrowRight");
-    cy.get("@changeSpy").should("have.been.calledWith", [1, 4, 7]);
-
-    cy.realPress("ArrowRight");
-    cy.get("@changeSpy").should("have.been.calledWith", [2, 5, 8]);
-
-    // Should not push beyond max
-    cy.realPress("ArrowRight");
-    cy.get("@changeSpy").should("have.callCount", 3);
-    cy.findAllByRole("slider").eq(0).should("have.attr", "aria-valuenow", "2");
-    cy.findAllByRole("slider").eq(1).should("have.attr", "aria-valuenow", "5");
-    cy.findAllByRole("slider").eq(2).should("have.attr", "aria-valuenow", "8");
-  });
-});
-
-describe("Given a non-pushable range slider", () => {
-  it("WHEN moving a handle, it should be constrained by the handles next to it", () => {
-    const changeSpy = cy.stub().as("changeSpy");
-    cy.mount(
-      <Slider
-        label={"TestLabel"}
-        min={-8}
-        max={8}
-        step={1}
-        pageStep={4}
-        defaultValue={[-1, 3, 7]}
-        onChange={changeSpy}
-      />
-    );
-
-    cy.findAllByRole("slider").should("have.length", 3);
-
-    cy.findAllByRole("slider").eq(0).focus();
-    cy.realPress("PageUp");
-    cy.realPress("ArrowUp");
-    cy.realPress("ArrowRight");
-    cy.realPress("End");
-    cy.get("@changeSpy")
-      .should("have.been.calledOnce")
-      .and("been.calledWith", [3, 3, 7]);
-
-    cy.findAllByRole("slider").eq(2).focus();
-    cy.realPress("Home");
-    cy.realPress("PageDown");
-    cy.realPress("ArrowLeft");
-    cy.realPress("ArrowDown");
-    cy.get("@changeSpy")
-      .should("have.been.calledTwice")
-      .and("been.calledWith", [3, 3, 3]);
+describe("Given a Slider", () => {
+  describe("Given a Slider with a single value", () => {
+    it("THEN it should have ARIA roles and attributes", () => {
+      cy.mount(
+        <Slider
+          style={{ width: "400px" }}
+          min={5}
+          max={125}
+          step={5}
+          defaultValue={100}
+        />
+      );
+      cy.findByRole("slider")
+        .should("have.attr", "aria-valuemin", "5")
+        .and("have.attr", "aria-valuemax", "125")
+        .and("have.attr", "aria-valuenow", "100");
+    });
+
+    it("THEN onChange should fire and thumb should move when mouse down on slider track", () => {
+      const changeSpy = cy.stub().as("changeSpy");
+      cy.mount(<Slider style={{ width: "400px" }} onChange={changeSpy} />);
+      cy.get(".saltSliderTrack").trigger("mousedown", {
```

Comment:
Some test using class getter some using role .?

Reply:
the slider track doesn't have a role attached so can't get by role